x-pack/plugins/osquery/public/packs/queries/ecs_mapping_editor_field.tsx

```diff
@@ -692,12 +691,12 @@ export const ECSMappingEditorForm = forwardRef<ECSMappingEditorFormRef, ECSMappi
       }
     }, [handleSubmit, formData, onAdd]);
 
-    useEffect(() => {
-      if (defaultValue) {
-        validate();
-        validateFields(['value.field']);
-      }
-    }, [defaultValue, osquerySchemaOptions, validate, validateFields]);
+    // useEffect(() => {
+    //   if (defaultValue) {
+    //     validate();
+    //     validateFields(['result.value']);
+    //   }
+    // }, [defaultValue, osquerySchemaOptions, validate, validateFields]);
```

Comment:
Should this code be deleted or is there a reason to keep it commented out? @sebelga

Reply:
That's the security team code so I left it as is. I just replaced the old `__validateFields` with the `validateFields` in the file. They might left it for a reason 😊